assets/templates/stat.html

```diff
@@ -1,43 +1,42 @@
-<div class="column-two-thirds">
-  <table>
-    <thead>
-      <tr>
-        <th class="centre_name" scope="col">{{name}}</th>
-        {{#beds}}
-        <th scope="col" class="numeric">{{type}} bed{{plurality}}</th>
-        {{/beds}}
-        <th class="numeric centre_updated" scope="col">Updated<br>{{updated_formatted}}</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr class="row_centre_capacity">
-        <th scope="row">Capacity</th>
-        {{#beds}}
-        <td class="numeric">{{capacity}}</td>
-        {{/beds}}
-        <td>&nbsp;</td>
-      </tr>
-      <tr class="row_centre_occupied">
-        <th scope="row">- Occupied</th>
-        {{#beds}}
-        <td class="numeric">{{occupied}}</td>
-        {{/beds}}
-        <td>&nbsp;</td>
-      </tr>
-      <tr class="row_centre_ooc">
-        <th scope="row">- Beds out of commission</th>
-        {{#beds}}
-        <td class="numeric">{{ooc}}</td>
-        {{/beds}}
-        <td>&nbsp;</td>
-      </tr>
-      <tr class="row_centre_availability">
-        <th scope="row">Availability</th>
-        {{#beds}}
-        <td class="numeric">{{available}}</td>
-        {{/beds}}
-        <td>&nbsp;</td>
-      </tr>
-    </tbody>
-  </table>
+<div class="grid-row">
+  <div class="column-three-quarters">
+    <h2 class="centre_name">{{name}}</h2>
```

Comment:
I was wrestling with the semantics of this, in terms of the visuals originally provided. Realistically, I couldn't meet the original design, without resorting to adding table cells merely for presentational use, which is nonsense.

For now, I can certainly render it as a caption, and leave the overall design as part of a discussion with Tom, and determine whether we retain the intended layout, or revise it accordingly.